### ISSUE TITLE
High contrast option

### ISSUE THREAD

**Issue by tamius-han** [Xbozon/rpg-styled-ui#63]
Hi. I'm really a fan of the look of this skin in general, but I have some minor complaints.

The theme seems a bit too dark with not enough contrast. This isn't a big problem when playing at night on a dark map, but under certain conditions, such as:

* the map is bright
* there's sun shining outside and it's shining right onto my monitor

this theme's lack of contrast makes things a bit hard to read. Few examples:

* In this example, there's no place on the map that would be darker than the text in either item window or the chat sidebar (ignore the orange bit and the font, i've got custom fonts everywhere and custom colors on classless h1-h6 tags):
![slika](https://user-images.githubusercontent.com/12505802/178136313-86b3d4ec-8cd5-4336-8edf-4bb20ec73223.png)

The leftmost window, for comparison, is tidy5e with dark mode. Thanks to the higher contrast, text is significantly more readable there.

Similar thing applies to journal, where the paper is rather dark by default:
![slika](https://user-images.githubusercontent.com/12505802/178140391-c6dff9de-b528-4f96-b807-76ac9ec23ca4.png)

I feel that brighter background and brighter text would make the theme much more readable:
![slika](https://user-images.githubusercontent.com/12505802/178141584-cb75af8b-cb85-4332-9765-c8dd992cb182.png)

**Comment by Sasmira** [comment]
all time the same problem :D
If you change something for one person, another person say that prefer the old version ... 

There are no really solution ... you can fix yourself this problem with a fork, stay with author values or try another Ui ... I'm not saying this to be mean and do nothing about it, it's just a redundant question that has no real solutions ... 

I say this from experience, because I have the same problem with Pathfinder Ui v3 which is a fork of RPG Styled Ui :)

**Comment by Xbozon** [comment]
@tamius-han 
I have to maintain a balance between visual quality and readability. Alas, the high contrast makes the interface not so attractive :(

I correct what I can, but it's difficult.

**Comment by tamius-han** [comment]
@Sasmira 

> There are no really solution ... you can fix yourself this problem with a fork,

Yes but without a proper defined license that's _technically_ of questionable legality. 

But doesn't hurt to ask so @Xbozon may I maintain a fork of this theme? (Even though I'll probably go slightly beyond merely upping the contrast a bit in mine)

**Comment by Xbozon** [comment]
@tamius-han 

Yes, you can create forks quite freely. I would appreciate it if you could post a link to the original repository and author. 
I'm still on the lookout for a fairly proper license, but in this case let's go with fair use :)

**Comment by tamius-han** [comment]
Okay, thanks.

**Comment by Xbozon** [comment]
@tamius-han 

I thought a little bit about the problem. It might be easier for you to release an addon that increases the contrast and install it as an additional mod. That way you'll get all the updates, but have your own changes.

Just enough to make a mod that imports another css file.

**Comment by bearsy1** [comment]
@tamius-han 

I like the idea of a higher contrast and easier readability while maintaining most of the aesthetics from the original! I tried installing your forked version but I'm not sure if I did it right, or if things are working properly though. 

I have the original module theme installed but it's not enabled in the settings, only the high contrast version is. Is this the correct way to use it? 

As for comparing how it looks -- I can't tell if things have changed or if it's changes are taking effect. Whether or not I enable the high contrast version, the original, or both at the same time, it doesn't seem to be visually different for me... not like the images you posted of your high contrast view. Any help/advice to get it looking like the third image you posted would be appreciated!
